confidential-data-hub/hub/src/bin/config/mod.rs

```diff
@@ -11,7 +11,13 @@ use log::{debug, warn};
 use serde::Deserialize;
 use tokio::fs;
 
-const DEFAULT_CDH_SOCKET_ADDR: &str = "unix:///run/confidential-containers/cdh.sock";
+cfg_if::cfg_if! {
+    if #[cfg(feature = "ttrpc")] {
+        const DEFAULT_CDH_SOCKET_ADDR: &str = "unix:///run/confidential-containers/cdh.sock";
+    } else {
+        const DEFAULT_CDH_SOCKET_ADDR: &str = "127.0.0.1:50000";
```

Comment:
It might be good to add a note to the docs that the gRPC feature will expose the API over TCP/IP socket rather than unix socket which might have security implications.

Reply:
Sounds good. Added in the `README.md`